modules/schema/schema.go

```diff
@@ -147,6 +147,17 @@ func getCollectionSchema(doc *ast.Document, dbName, collectionName string) (Sche
 						fieldTypeStuct.IsCreatedAt = true
 					case directiveUpdatedAt:
 						fieldTypeStuct.IsUpdatedAt = true
+					case directiveDefault:
+						fieldTypeStuct.IsDefault = true
+
+						for _, arg := range directive.Arguments {
+							switch arg.Name.Value {
+							case "value":
+								val, _ := utils.ParseGraphqlValue(arg.Value, nil)
+
+								fieldTypeStuct.Default = val
+							}
+						}
```

Comment:
Throw error if value isn't set after the loop ends

Reply:
@YourTechBud i think we should check if the default field is set or not. which is an interface after the loop ends.
by the way where is the **value** filled is created/initialized